an-der-uni/ausland.tex

```diff
@@ -9,5 +9,7 @@
 
 AuslandsBAföG müsst ihr übrigens separat beantragen, die Zu\-stän\-dig\-keit dafür hängt vom jeweiligen Ausland ab, eine Liste der zuständigen Ämter findet man z.B. unter \url{AuslandsBAföG.de}. In Heidelberg kann man sich auch direkt an das Dezernat Internationale Beziehungen (früher Akademisches Auslandsamt, AAA)\footnote{\url{http://www.uni-heidelberg.de/studium/kontakt/auslandsamt/}} wenden, das ihr in der Seminarstraße 2 findet. Ihr müsst dann ins Zimmer 139.
 
+Sehr wichtig ist noch zu wissen, dass man Urlaubssemester beantragen kann, damit die Fachsemesterzahl (vorallem wichtig für alle, die Inlands-BAföG erhalten) nicht weiterläuft. Dafür läuft die Hochschulsemesterzahl weiter. Man kann sich dann die im Ausland erworbenen CPs (nach Absprache) dann anrechnen lassen.
```

Comment:
Für das Urlaubssemester ist laut Webseite der Uni die Studierendenadministration in der Seminarstraße 2 zuständig. Mehr Infos und das Formular zum Urlaubssemester ist zu finden unter:
https://www.uni-heidelberg.de/studium/imstudium/formalia/beurlaubung.html